src/fetcher/serlo-api.tsx

```diff
@@ -39,7 +40,15 @@ export default async function fetchContent(alias: string, redirect) {
       return { redirect: reqData.uuid.alias }
     }
     const contentId = reqData.uuid.id
-    return { contentId, alias, ...processResponse(reqData) }
+
+    const processed = processResponse(reqData)
+    const processedData = processed.data as any
```

Comment:
*sad face* 😉, should probably refactor / type / unit test that as a follow-up sometime.

Reply:
yeah… I know.
and the really sad thing is that even that hack took me some time to figure out. 
TS is great in making me feel stupid, that's for sure :) if you have time and hints how I could fix it let me know.

Reply:
I'm fine with leaving that at it is for now. My PR #267 showed that there are many places where the typing is `any` and we should go over that sometime. This particular line would probably okay if `processResponse` was properly typed.